autogen/oai/telemetry.py

```diff
@@ -0,0 +1,91 @@
+import sqlite3
+import datetime
+import json
+
+try:
+    import openai
+    from openai.types.chat import ChatCompletion
+    ERROR = None
+except ImportError:
+    ERROR = ImportError("Please install openai>=1 and diskcache to use autogen.OpenAIWrapper.")
+
+
+class Telemetry:
+    def __init__(self):
+        self.con = sqlite3.connect("telemetry.db")
+        self.cur = self.con.cursor()
+        query = """
+            CREATE TABLE IF NOT EXISTS messages(
```

Comment:
I am wondering if we should also have maybe a session_id  or process_id field. If we are always writing to the same database file, it might be hard to distinguish between execution runs -- especially when doing active iterative development. 

A session_id can be as simple as a global uuid field created in memory when the library is loaded or something, so that it's unique for each python invocation, but consistent across conversations occurring in that execution.